pull-request.sh

```diff
@@ -90,6 +91,21 @@ main () {
     fi
     echo "Pull requests will go to ${PULL_REQUEST_BRANCH}"
 
+    if [ -z "${PULL_REQUEST_DRAFT}" ]; then
+        echo "No explicit preference for draft PR: created PRs will be normal PRs."
+        PULL_REQUEST_DRAFT="false"
+    else
+        echo "Environment variable PULL_REQUEST_DRAFT set to a value: created PRs will be draft PRs."
+        PULL_REQUEST_DRAFT="true"
+    fi
+
+    # separate block for logging draft vs not draft
+    if [ "$PULL_REQUEST_DRAFT" = "true" ]; then
+        echo "Created PRs will be draft PRs."
+    else 
+        echo "Created PRs will be normal PRs."
+    fi
+
```

Comment:
I think you might have missed a save again? Probably just after nixing 101-107 here